.github/workflows/cd.yml

```diff
@@ -369,6 +379,49 @@ jobs:
           artifactErrorsFailBuild: true
           bodyFile: ./CHANGELOG.md
 
+      - name: Update Preview TTK ai Key
+        if: ${{ github.event_name == 'schedule' || (contains(steps.version-change.outputs.CHANGED, 'ms-teams-vscode-extension@') && github.event.inputs.isPreview != 'no')}}
```

Comment:
What about the case of manual run? Will the replacement of preview ai key be missed in that case?

Reply:
no, once manual run, the inputs.isPreview should not be 'no', then will trigger the preview workflow